test/functional/fixtures/api/es-next/press-key/testcafe-fixtures/press-key-test.js

```diff
@@ -0,0 +1,20 @@
+// NOTE: to preserve callsites, add new tests AFTER the existing ones
+import { Hybrid } from 'testcafe';
+import { expect } from 'chai';
+
+
+fixture `PressKey`
+    .page `http://localhost:3000/api/es-next/press-key/pages/index.html`;
+
+const focusInput    = Hybrid(() => document.getElementById('input').focus());
+const getInputValue = Hybrid(() => document.getElementById('input').value);
+
+test('Press keys', async t => {
+    await focusInput();
```

Comment:
I think we can add `t.focus()` and I think `pressKey` without selector is conveniently, because activeElement can be changed during execution key sequence.